consensus/reactor_test.go

```diff
@@ -663,7 +635,7 @@ func timeoutWaitGroup(t *testing.T, n int, f func(int), css []*State) {
 
 	// we're running many nodes in-process, possibly in in a virtual machine,
 	// and spewing debug messages - making a block could take a while,
-	timeout := time.Second * 300
+	timeout := time.Second * 120
```

Comment:
why change?

Reply:
My feeling is that it's way too excessive and it's annoying if your writing a test and there is a bug because you have to wait 5 minutes to see what went wrong. I can change it back if you feel 5 minutes is necessary for the slow CI - What do you think?

Reply:
In any case, this is unrelated to the evidence pool change - please open a separate PR.